src/class-parsely.php

```diff
@@ -287,8 +287,9 @@ public static function post_has_trackable_status( $post ): bool {
 		 * @param string[]    $trackable_statuses The list of post statuses that are allowed to be tracked.
 		 * @param int|WP_Post $post               Which post object or ID is being checked.
 		 */
-		$statuses          = apply_filters( 'wp_parsely_trackable_statuses', array( 'publish' ), $post );
-		$cache[ $post_id ] = in_array( get_post_status( $post ), $statuses, true );
+		$statuses = apply_filters( 'wp_parsely_trackable_statuses', array( 'publish' ), $post );
+
+		$cache[ $post_id ] = in_array( get_post_status( $post ), $statuses, true ) && ! post_password_required( $post );
```

Comment:
I'd probably move the `post_password_required` to the top of the function since there's no need to fire the `wp_parsely_trackable_statuses` filters if they've no chance of passing the check on protected posts.